juntagrico/entity/member.py

```diff
@@ -71,6 +72,19 @@ class Member(JuntagricoBaseModel):
 
     objects = MemberQuerySet.as_manager()
 
+    previous_email = None
+
+    @staticmethod
+    def post_save(sender, instance, created, **kwargs):
+        if instance.previous_email != instance.email or created:
```

Comment:
Juntagrico already stores a copy of the old instance in `instance._old`. You can get the previous email with `instance._old['email']`

Reply:
thanks for the input! fixed.